src/app/productConfig/productConfigForm/productConfigForm.tpl.html

```diff
@@ -21,14 +21,6 @@ <h4 id="product-config-header" class="give-gift-header" translate>
     </div>
   </div>
 
-  <loading ng-if="$ctrl.loading">
-    <translate>{{'LOADING_GIFT_DETAILS'}}</translate>
-  </loading>
-
-  <div ng-if="$ctrl.errorLoading" class="alert alert-danger" role="alert">
-    <p translate='LOADING_ERROR' translate-values="{loadData: '$ctrl.loadData()'}" translate-compile></p>
-  </div>
```

Comment:
Taking this out of here will remove the loading indicator from the standard checkout